packages/esm-patient-list-app/src/index.ts

```diff
@@ -36,7 +46,7 @@ function setupOpenMRS() {
     },
     {
       path: `${spaBasePath}/:uuid?`,
-      title: ([x]) => `${x}`,
+      title: ([x]) => getName(`${x}`),
```

Comment:
```suggestion
      title: ([patientListUuid]) => getListName(patientListUuid),
```

Reply:
`patientListUuid` is more appropriate here. Also, `patientListUuid` is a string, so there's no need to interpolate it.